test/testBlocks.jl

```diff
@@ -795,8 +795,8 @@ function VSDTestBlock!(fg, v1)
     @test mergeVariableState!(fg, :a, vnd) == 1
 
     # Bulk copy update x0
-    @test updateVariableSolverData!(fg, [v1], :default) == nothing
-
+    @test DFG.mergeVariableStates!(fg, [:a=>vnd]) == 1
```

Comment:
Yes on using verb `merge` and dropping `update`, see:
- #1085

Reply:
yes, merge is what we decided. Captured in [wiki](https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/wiki/Standard-Definitions-of-API-via-verbNoun-structure)